server/cabal.project

```diff
@@ -35,8 +35,8 @@ package graphql-engine
 
 source-repository-package
   type: git
-  location: https://github.com/hasura/pg-client-hs.git
-  tag: 70a849d09bea9461e72c5a5bbde06df65aab61c0
+  location: https://github.com/abooij/pg-client-hs.git
```

Comment:
Let us merge your changes to `pg-client-hs` master and update the reference here to `hasura`. Please link to the PR in pg-client-hs repo.

Reply:
Hi Rakesh! I've linked to this PR above, and I have now also requested you as a reviewer for that PR.

Reply:
Hi @abooij , I merged the PR in pg-client-hs repo. Please refer the new commit. 
Thank you.